### ISSUE TITLE
Enhance Read/Write attributes to support multiple attributes at once

### ISSUE THREAD

**Issue by cdjackson** [zsmartsystems/com.zsmartsystems.zigbee#529]
These two commands support reading and writing multiple attributes at once, but this isn't supported in ```ZclCluster``` class

**Comment by cdjackson** [comment]
This is partially implemented (for reading attributes) in #530

**Comment by cdjackson** [comment]
Writing attributes is added in #527 

```
    /**
     * Writes a number of attributes in a single command
     * 
     * @param attributes a Map of {@link ZclAttribute}s and their values
     * @return command future {@link CommandResult}
     */
    public Future<CommandResult> writeAttributes(final Map<ZclAttribute, Object> attributes);

    /**
     * Writes a number of attributes in a single command
     *
     * @param attributes a List of {@link WriteAttributeRecord}s with the attribute ID, type and value
     * @return command future {@link CommandResult}
     */
    public Future<CommandResult> writeAttributes(List<WriteAttributeRecord> attributes);
```
